src/xdocs/config.xml

```diff
@@ -952,30 +957,18 @@
 
     </section>
 
-    <section name="XML Structure">
+    <section name="XML Configuration Structure">
 
-      <b>Configuration XML Document</b>
       <p>
-        The following DTD for a configuration XML document specifies the
-        hierarchy of modules and their properties:
+        DTD is used to validate
```

Comment:
`to validate` => `to validate the`

Reply:
done

Reply:
strange. I did it. 
done.
verified after push to remote